web/templates/datasets/dataset.html

```diff
@@ -485,4 +474,69 @@ <h2 class="card-title">Logbook</h2>
             </div>
         </div>
     </div>
+    {% if request.user.can_publish %}
+    <div class="row mt-4">
+        {#    Exposure endpoints for the DATASET  #}
+        <div class="card with-max-height col-md-12 ml-0">
+            <div class="card-body">
+                {% if request.user.can_publish %}
```

Comment:
Sorry, my last comment should have been clearer.

I meant that the `can_edit` was not useful, as the more restrictive `can_publish` is already evaluated above (line 477). I think you can just remove the condition in the card body